swaggen/src/main/java/domain/output/PathURL.java

```diff
@@ -0,0 +1,45 @@
+package domain.output;
+
+/**
+ * The path and base path of the server
+ */
+public class PathURL {
```

Comment:
I just noticed this, can we rename this to PathUri. PathURL does not follow standard naming conventions. Furthermore, the use of "url" versus "uri" is still an open debate. I don't care which one we use, but let's keep it consistent.

Reply:
URI is more accurate since the URL would refer to the server URL (there's another PR in regards to this)

Reply:
That is not entirely accurate, but a debate I don't care to have. `URI` is definitely applicable, while `URL` may be applicable. Let's go with `URI` from now on and be consistent about it.